pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -159,21 +165,11 @@ func (gs *GPUDevices) AddResource(pod *v1.Pod) {
 	for _, val := range podDev {
 		for _, deviceused := range val {
 			for index, gsdevice := range gs.Device {
-				if gsdevice.UUID == deviceused.UUID {
-					klog.V(4).Infoln("VGPU recording pod", pod.Name, "device", deviceused)
-					gs.Device[index].UsedMem += uint(deviceused.Usedmem)
-					gs.Device[index].UsedNum++
-					gs.Device[index].UsedCore += uint(deviceused.Usedcores)
-					_, ok := gs.Device[index].PodMap[pod.Name]
-					if !ok {
-						gs.Device[index].PodMap[pod.Name] = &GPUUsage{
-							UsedMem:  0,
-							UsedCore: 0,
-						}
+				if strings.Contains(deviceused.UUID, gsdevice.UUID) {
+					err := gs.Sharing.AddPod(gsdevice, uint(deviceused.Usedmem), uint(deviceused.Usedcores), string(pod.UID), deviceused.UUID)
```

Comment:
Usedmem and Usedcores are already uint type, no need to covert.

Reply:
Done